S4TF_Tutorial_2.ipynb

```diff
@@ -275,7 +275,7 @@
         "  </td></tr>\n",
         "</table>\n",
         "\n",
-        "Swift for TensorFlow has many [optimization algorithms](https://github.com/rxwei/DeepLearning/blob/master/Sources/DeepLearning/Optimizer.swift) available for training. This model uses the Adam optimizer that implements the *[Adam Algorithm](http://arxiv.org/abs/1412.6980). \n",
+        "Swift for TensorFlow has many [optimization algorithms](https://github.com/rxwei/DeepLearning/blob/master/Sources/DeepLearning/Optimizer.swift) available for training. This model uses the Adam optimizer that implements the *[Adam Algorithm](http://arxiv.org/abs/1412.6980)*. \n",
         "\n",
         "During training, the optimizer function is used to calculate adjustments to the model's internal variables. The goal is to adjust the internal variables until the model (which is really a math function) mirrors the actual equation for converting Celsius to Fahrenheit.\n",
         "\n",
```

Comment:
Below this, "optimizer" should probably not be capitalized. The sentence "Too large, and accuracy goes down." could maybe be improved with "Too large, and accuracy may decrease or training could diverge entirely."